miio/cooker.py

```diff
@@ -0,0 +1,706 @@
+import enum
+import logging
+import string
+from collections import defaultdict
+from datetime import time
+from typing import Optional
+
+import click
+
+from .click_common import command, format_output, EnumType
+from .device import Device, DeviceException
+
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_PRESSURE1 = 'chunmi.cooker.press1'
+MODEL_PRESSURE2 = 'chunmi.cooker.press2'
+MODEL_NORMAL1 = 'chunmi.cooker.normal1'
+MODEL_NORMAL2 = 'chunmi.cooker.normal2'
+MODEL_NORMAL4 = 'chunmi.cooker.normal3'
+MODEL_NORMAL3 = 'chunmi.cooker.normal4'
+MODEL_NORMAL5 = 'chunmi.cooker.normal5'
+
+MODEL_PRESSURE = [MODEL_PRESSURE1, MODEL_PRESSURE2]
```

Comment:
If MODEL_PRESSURE{1,2} are used nowehere else than here, better just simply use them instead of exposing a global variable.

Reply:
These consts are important in future for all the edge-cases.